Plugins/Flow.Launcher.Plugin.Url/Languages/en.xaml

```diff
@@ -1,18 +1,22 @@
-﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:system="clr-namespace:System;assembly=mscorlib">
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:system="clr-namespace:System;assembly=mscorlib">
 
-    <system:String x:Key="flowlauncher_plugin_url_open_search_in">Open search in:</system:String>
-    <system:String x:Key="flowlauncher_plugin_new_window">New Window</system:String>
-    <system:String x:Key="flowlauncher_plugin_new_tab">New Tab</system:String>
-    
     <system:String x:Key="flowlauncher_plugin_url_open_url">Open url:{0}</system:String>
     <system:String x:Key="flowlauncher_plugin_url_cannot_open_url">Can't open url:{0}</system:String>
 
     <system:String x:Key="flowlauncher_plugin_url_plugin_name">URL</system:String>
     <system:String x:Key="flowlauncher_plugin_url_plugin_description">Open the typed URL from Flow Launcher</system:String>
 
-    <system:String x:Key="flowlauncher_plugin_url_plugin_set_tip">Please set your browser path:</system:String>
     <system:String x:Key="flowlauncher_plugin_url_plugin_choose">Choose</system:String>
     <system:String x:Key="flowlauncher_plugin_url_plugin_filter">Application(*.exe)|*.exe|All files|*.*</system:String>
+
+    <system:String x:Key="flowlauncher_plugin_url_use_custom_browser">Use custom web browser instead of Flow default web browser</system:String>
```

Comment:
```suggestion
    <system:String x:Key="flowlauncher_plugin_url_use_custom_browser">Use custom instead of Flow's default web browser</system:String>
```